vta/src/dpi/tsim.cc

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
move to vta/hardware/dpi/tsim_device.cc

Although this may sound counter-intuitive, this is a module linked via simulator hardware, rather than the TVM itself.